File: LeetCode-Problems/Question/XOR.cpp
#include<bits/stdc++.h>

using namespace std;

int main()
{
    int numberOne, numberTwo;
    cout<<"Enter two numbers: ";
    cin>>numberOne>>numberTwo;
    int XOR=numberOne ^ numberTwo; 
    cout<<XOR<<endl;    
    // double power= 2**2; Doesn't work in c++.
}

// XOR(^) 
// Same numbers -> 0
//Different Numbers -> 1

File: LeetCode-Problems/Question/XOR.java
import java.util.*;

class XOR
{
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter two nos: ");
        int n=sc.nextInt();
        int m=sc.nextInt();

        int XOR= n^ m; //Still giving XOR
        System.out.println(XOR);
        // XOR= n ** m; //Doesn't work.
        System.out.println(XOR);
    }
}